src/tests/TestXMLTools/TestXMLTools.cpp

```diff
@@ -0,0 +1,223 @@
+// Copyright 2005-2017 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+// Include murmur_pch.h for STACKVAR
+#include "murmur_pch.h"
+
+#include <QtCore>
+#include <QtTest>
+
+#include "XMLTools.h"
```

Comment:
Please re-arrange so that Mumble and Qt includes are separated.
If XMLTools.h depends on Qt includes, it should probably include them itself.